src/CMakeLists.txt

```
@@ -26,6 +26,8 @@ FUNCTION(PODIO_ADD_LIB_AND_DICT libname headers sources selection )
   target_include_directories(${libname} PUBLIC
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+  )
+  target_include_directories(${libname} SYSTEM PUBLIC
```

```suggestion
    SYSTEM PUBLIC
```
Does this work with one call? (Mainly out of interest, rather than as an actual suggestion for fixing here)